scrapper.py

```diff
@@ -83,9 +109,31 @@ def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, 'r') as crawler:
+        data = json.loads(str(crawler.read()))
+        validating_seed_urls = data["base_urls"]
+        validating_max_articles = data["total_articles_to_find_and_parse"]
+        validating_max_articles_per_seed = data["max_number_articles_to_get_from_one_seed"]
+
+    if not isinstance(validating_seed_urls, list) or not isinstance(validating_max_articles, int)\
+            or not isinstance(validating_max_articles_per_seed, int):
+        raise UnknownConfigError
+
+    for url in validating_seed_urls:
+        response = requests.get(url)
```

Comment:
could we make a pre-flight check first? maybe a regex? or some conditions? you should not make any real requests at this point (!)